tools/runners/Icarus.py

```diff
@@ -4,9 +4,10 @@
 class Icarus(BaseRunner):
     def __init__(self):
         super().__init__("icarus")
+        self.ex = 'iverilog'
```

Comment:
Can we rename `ex` to something self-explanatory?

Reply:
`executable`?

Reply:
+1

A comment about this in the main class would be nice too

Reply:
Maybe the executable could also be a kwarg in the main class constructor?

Reply:
So you could just call `super().__init__("icarus", "iverilog")` etc